### PR TITLE
[Datafactory] add missing ssis entity types

### PR DESCRIPTION
**Pull request by shawncx**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json
Comment (nschonni) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json]:
```suggestion
          "description": "Whether parameter is required."
```

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json
Comment (nschonni) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json]:
```suggestion
          "description": "Whether variable is sensitive."
```

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json
Comment (zhangyd2015) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json]: The changes also need to be in version 2017-09-01-preview.

**Comment by hvermis** [comment]
@shawncx Before you merge this let's discuss this internally.

**Review by sarangan12 — COMMENTED** [review]
Looks fine. Please fix the minor typos before I merge. 

Also, when you are ready to merge, Please remove [Do not merge] from the heading.

**Review by sarangan12 — APPROVED** [review]

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json
Comment (hvermis) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json]: @zhangyd2015  The changes in private swagger not betting regenerated into SDKs, so no need to change there.

**Comment by sarangan12** [comment]
Per ARM Guidelines, since you are adding new models, this needs ARM Feedback....

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json
Comment (antmarti-microsoft) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json]: I can see you're adding new object definitions, but they don't appear to be referenced by any of the existing properties in this swagger. How are these new objects used?

Comment (shawncx) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json]: `SsisFolder`, `SsisProject`, `SsisPackage` and `SsisEnvironment` are subclass of `SsisObjectMetadata`. [IntegrationRuntimeObjectMetadata_Get](https://github.com/Azure/azure-rest-api-specs/blob/16a2ac3dfd26f79a49488411fa80734419763984/specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json#L1420) returns `SsisObjectMetadata` list, the elements belong to one of the subclass.
By adding these  subclass, customer can get the particular properties by casting. The elements in list must belong to same subclass.

**Comment by hvermis** [comment]
@antmarti-microsoft @sarangan12 Please proceed with review of this PR.

**Comment by ravbhatnagar** [comment]
This has been reviewed by Anthony from ARM side and the comment from him has been addressed. Signing off from ARM side.

**Comment by shawncx** [comment]
@hvermis Seems this pr is blocked... is there anything is should do?

**Comment by hvermis** [comment]
@sarangan12 Can you please merge this PR? ARM signed off.

**Pull request merged**
